Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -558,7 +558,7 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayPrototype::reverse)
     if (array->indexed_properties().is_empty())
         return array;
 
-    Vector<Value> array_reverse;
+    MarkedValueList array_reverse{array->heap()};
```

Comment:
```suggestion
    MarkedValueList array_reverse { array->heap() };
```

at least I think that's what clang-format should do?

Reply:
it should lol